discovery/seata-discovery-consul/src/main/java/io/seata/discovery/registry/consul/ConsulRegistryServiceImpl.java

```diff
@@ -268,10 +268,10 @@ private NewService createService(InetSocketAddress address) {
      * @param cluster
      */
     private void refreshCluster(String cluster) {
-        if (cluster == null) {
+        if (StringUtils.isBlank(cluster)) {
             return;
         }
-        Response<List<HealthService>> response = getHealthyServices(getClusterName(), -1, -1);
+        Response<List<HealthService>> response = getHealthyServices(cluster, -1, -1);
```

Comment:
add some comment `getClusterName` only for server

Reply:
done